scipy/spatial/distance.py

```diff
@@ -43,23 +43,29 @@
    cityblock        -- the Manhattan distance.
    correlation      -- the Correlation distance.
    cosine           -- the Cosine distance.
-   dice             -- the Dice dissimilarity (boolean).
    euclidean        -- the Euclidean distance.
-   hamming          -- the Hamming distance (boolean).
-   jaccard          -- the Jaccard distance (boolean).
-   kulsinski        -- the Kulsinski distance (boolean).
    mahalanobis      -- the Mahalanobis distance.
-   matching         -- the matching dissimilarity (boolean).
    minkowski        -- the Minkowski distance.
-   rogerstanimoto   -- the Rogers-Tanimoto dissimilarity (boolean).
-   russellrao       -- the Russell-Rao dissimilarity (boolean).
    seuclidean       -- the normalized Euclidean distance.
-   sokalmichener    -- the Sokal-Michener dissimilarity (boolean).
-   sokalsneath      -- the Sokal-Sneath dissimilarity (boolean).
    sqeuclidean      -- the squared Euclidean distance.
    wminkowski       -- the weighted Minkowski distance.
-   yule             -- the Yule dissimilarity (boolean).
+   
+Similarly, distance functions between two boolean vectors (representing sets)
```

Comment:
Using the word `Similarly` here may lead to confusion: almost all of the binary distances are derived from some form of similarity measure, and an unattentive reader may understand that it is the similarity that is being calculated. Using `Also available are`, or some other similar ;-) expression may be a better option.